tools/py/neb_interpolate.py

```diff
@@ -344,6 +368,9 @@ def spline_resample(q, nbeads_old, nbeads_new, k=3):
     # Below this point, it shouldn't matter where the inputs came from.
     # =================================================================
     print("Imported structures: %d geometries, %d atoms each." % (nbeads_old, natoms))
```

Comment:
Could the print statements be consistent with the i-pi output like "@NEB: Importer structure"

Reply:
Hi Venkat! I'm not sure about the @NEB prefix, because it's a standalone utility which is not meant to be called from the main code. I think it is a reasonable suggestion in the `spline_resample()` because it is designed to be reusable, but not in `__main__()`.